.github/CODEOWNERS

```diff
@@ -1,10 +1,14 @@
 # Code generated by githubgen. DO NOT EDIT.
 #####################################################
 #
-# List of codeowners
+# List of codeowners for OpenTelemetry Collector Contrib
```

Comment:
Is this change auto-generated?

Reply:
These changes we generated when I executed following command `.tools/githubgen -skipgithub`.